src/main/java/com/gtnewhorizons/modularui/common/widget/FluidSlotWidget.java

```diff
@@ -282,12 +299,16 @@ public static boolean fluidChanged(@Nullable FluidStack current, @Nullable Fluid
                 || (current != null && (current.amount != cached.amount || !current.isFluidEqual(cached)));
```

Comment:
Remove this method

Reply:
which is this method?

Reply:
`fluidChanged(FluidStack, FluidStack)`

Reply:
ah. I wasn't sure if I could delete it as it was a public method, but done